audit/projects/k8s-infra-ii-sandbox/services/bigquery/bigquery.datasets.json

```diff
@@ -44,24 +44,6 @@
     },
     "location": "US"
   },
-  {
-    "kind": "bigquery#dataset",
-    "id": "k8s-infra-ii-sandbox:etl_script_generated_set_20210810",
-    "datasetReference": {
-      "datasetId": "etl_script_generated_set_20210810",
-      "projectId": "k8s-infra-ii-sandbox"
-    },
-    "location": "US"
-  },
```

Comment:
@BobyMCbobs two questions:
- do you know why this is happening?
- I'm noticing there are a lot of generated data sets here, do we really need all of them?

Reply:
Ref: https://github.com/kubernetes/k8s.io/issues/4402. 

Cleanup done on this project.